Assets/Scripts/EnemySpawner.cs

```diff
@@ -0,0 +1,83 @@
+using System.Collections;
+using UnityEngine;
+
+namespace FunMath
+{
+    public class EnemySpawner : MonoBehaviour
+    {
+        public GameObject enemyPrefab; // Reference to the enemy prefab
```

Comment:
@ShumWengSang, isn't there a way to make the `enemyPrefab` required so Laura doesn't have to do null checks for it later? I feel like you had that setting on the class. Like `[RequiresComponent("enemyPrefab")]` or something.

Reply:
Yeah that ensures that in editor time at least, there is a component on the object that is required. It doesn't work if an component is added on to the gameobject iirc